.github/workflows/releasability-check.yml

```diff
@@ -13,6 +13,6 @@ jobs:
     permissions:
       id-token: write
       contents: write
-    uses: SonarSource/gh-action_release/.github/workflows/releasability-check.yaml@5.3.2
+    uses: SonarSource/gh-action_release/.github/workflows/releasability-check.yaml@5
```

Comment:
Should be v5

Reply:
Git tag is [not](https://github.com/SonarSource/gh-action_release/releases) prefixed with `v` for this action. I'm not sure if Github can deduce it automatically, but it definitely can match tag without a `v`